include/deal.II/fe/fe_dgq.h

```diff
@@ -451,7 +451,53 @@ class FE_DGQArbitraryNodes : public FE_DGQ<dim, spacedim>
  * See the base class documentation in FE_DGQ for details.
  */
 template <int dim, int spacedim = dim>
-class FE_DGQLegendre : public FE_DGQ<dim, spacedim>
+class FE_DGQArbitraryPolynomialSpace : public FE_DGQ<dim, spacedim>
```

Comment:
I've got to admit that this looks a bit backward. `FE_DGQ` should be the *derived* class, and `FE_DGQArbitraryPolynomialSpace` the base class. Do you know why it is this way around?